google/cloud/grpc_utils/async_operation.h

```diff
@@ -16,7 +16,13 @@
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_GRPC_UTILS_ASYNC_OPERATION_H
 
 #include "google/cloud/async_operation.h"
-#include "google/cloud/version.h"
+#include "google/cloud/grpc_utils/version.h"
```

Comment:
why did this `#include` change, but the others didn't?

Reply:
Fixed.